src/screens/StarterPack/Wizard/State.tsx

```diff
@@ -73,7 +74,7 @@ function reducer(state: State, action: Action): State {
       updatedState = {...state, description: action.description}
       break
     case 'AddProfile':
-      if (state.profiles.length >= 51) {
+      if (state.profiles.length > STARTER_PACK_MAX_SIZE) {
         Toast.show(
           msg`You may only add up to 50 profiles`.message ?? '',
```

Comment:
Would it be feasible to amend this toast to refer to the `STARTER_PACK_MAX_SIZE`?

Reply:
Nice, thanks!